incubator/nodejs-kafka/README.md

```diff
@@ -0,0 +1,88 @@
+# Node.js Kafka Stack
+
+The Node.js Kafka stack extends the [Node.js stack](../nodejs/README.md) and provides a consistent way of developing applications using [node-rdkafka](https://www.npmjs.com/package/node-rdkafka) and [Express](http://expressjs.com).
+
+This stack is based on `Node.js v12` and `node-rdkafka v2.7.x` and enables health checking and application metrics out of the box. The stack also provides a set of tests written using the [Mocha](https://mochajs.org) test framework, while the application metadata is provided via a `package.json` file.
```

Comment:
It's probably not an immediate issue as this is using Node.js 12, but I recently came across `node-rdkafka` as somebody raised an issue in nodejs/node-gyp (https://github.com/nodejs/node-gyp/issues/2055) as it wasn't compiling with Node.js 13. I didn't do a deep search (just enough to demonstrate it wasn't a node-gyp issue) and found https://github.com/Blizzard/node-rdkafka/issues/733.

Reply:
Thanks Richard. I think we're fine to use this library on Node.js 12 for now.

Reply:
I got it building on 13.x. This is still incubator, and we're doing some ongoing research into what the "best" Kafka library is a the moment. The situation is fluid :-).